lib/js_rails_routes/generator.rb

```diff
@@ -18,14 +18,21 @@ class Generator
 
     include Singleton
 
-    attr_accessor :include_paths, :exclude_paths, :include_names, :exclude_names, :exclude_engines, :output_dir
+    attr_accessor :include_paths,
+                  :exclude_paths,
+                  :include_names,
+                  :exclude_names,
+                  :exclude_engines,
+                  :output_dir,
+                  :lower_camelcase
 
     def initialize
       self.include_paths   = /.*/
       self.exclude_paths   = /^$/
       self.include_names   = /.*/
       self.exclude_names   = /^$/
       self.exclude_engines = /^$/
+      self.lower_camelcase = false
```

Comment:
Please rename as `camelize` and take `"lower"`, `"upper"` and `nil`

Reply:
Why does it variable type is the String for `lower` and `upper`? It might be a good idea to use the Symbol because it never used the variable as String.

Reply:
I'd like to use as Symbol in Rakefile.
```
self.camelize = :lower
```

Reply:
make sense 👍